src/__tests__/split-by.test.js

```diff
@@ -0,0 +1,20 @@
+/* eslint-env node, jest */
+const { splitBy, asyncSplitBy, asyncToArray } = require('..')
+
+describe('splitBy', function () {
+  it('splitBy', function () {
+    const [a, b, c] = splitBy('AAABBCCCCD')
```

Comment:
Would be super helpful as a reviewer to have at very least a test that demonstrates the basic use case. It documents the function for me, and when I see the green flag from Travis I know that the code I'm reading actually does what it claims to.

Reply:
yes, sorry. I won't solicit feed backs anymore without tests